api/v1/azureappconfigurationprovider_types.go

```diff
@@ -30,7 +30,9 @@ import (
 type AzureAppConfigurationProviderSpec struct {
 	// The endpoint url of AppConfiguration which should sync the configuration key-values from.
 	// +kubebuilder:validation:Format=uri
-	Endpoint                  *string                                  `json:"endpoint,omitempty"`
+	Endpoint *string `json:"endpoint,omitempty"`
```

Comment:
nit: indentation

Reply:
There's `kubebuilder` notation between lines which influences `go fmt` result and the indention here.